workers/common/src/main/kotlin/common/Extensions.kt

```diff
@@ -42,18 +42,18 @@ val logger: Logger = LoggerFactory.getLogger(ConfigManager::class.java)
  * an exception, but the default file is allowed to not exist. Not being able to deserialize the file to the return type
  * [T] always leads to an exception.
  */
-inline fun <reified T> ConfigManager.readConfigFileWithDefault(
+inline fun <reified T> ConfigManager.deserializeConfigFileWithDefault(
```

Comment:
Commit message: "that *they* not only read"

Reply:
IMO "deserialize" is an awfully technical term for such a function name... even KxS itself uses "decode" instead. Better naming pairs could be e.g. "decode" / "read", "parse" / "load" etc.

Reply:
How about "readConfigFileValueWithDefault"?

Reply:
Fine with me.